contracts/multisig-prover/src/migrate.rs

```diff
@@ -0,0 +1,143 @@
+use axelar_wasm_std::{hash::Hash, FnExt, MajorityThreshold};
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{from_slice, Addr, DepsMut, Response, Uint256};
+use multisig::key::KeyType;
+use router_api::ChainName;
+
+use crate::{
+    encoding::Encoder,
+    state::{Config, CONFIG},
+};
+
+#[cw_serde]
+pub struct OldConfig {
+    pub admin: Addr,
+    pub governance: Addr,
+    pub gateway: Addr,
+    pub multisig: Addr,
+    pub coordinator: Addr,
+    pub service_registry: Addr,
+    pub voting_verifier: Addr,
+    pub destination_chain_id: Uint256,
+    pub signing_threshold: MajorityThreshold,
+    pub service_name: String,
+    pub chain_name: ChainName,
+    pub worker_set_diff_threshold: u32,
+    pub encoder: Encoder,
+    pub key_type: KeyType,
+}
```

Comment:
We should probably put this in some sort of folder or namespace or something to denote this struct represents the config from version 0.2. Or just call it Config0.2 or something. Otherwise what will we do when we need to add a new field to the config and perform a similar migration again?

Reply:
instead of renaming I moved the migration to migrate::v_0_3, let me know what you think